src/uiHelpers_nanos.c

```diff
@@ -0,0 +1,131 @@
+#include <os_io_seproxyhal.h>
+#if defined(TARGET_NANOS)
+#include "uiHelpers.h"
+#include "assert.h"
+#include "ux.h"
+#include "assert.h"
+#include "io.h"
+#include "utils.h"
+#include "securityPolicy.h"
+
+
+#ifdef HEADLESS
+#define HEADLESS_UI_ELEMENT() \
+    { \
```

Comment:
Hmm, did I really have this nested structure for headless element? Seems weird to me ... Actually -- from where is this code copied? I don't see it being deleted anywhere

Reply:
Oh, I see now. We are not using UI_TEXT macro here so that is why it feels different.